test/unit/vc2013/unit.vcxproj

```diff
@@ -271,7 +270,8 @@
     <ClCompile Include="..\src\SystemTest.cpp" />
     <ClCompile Include="..\src\TestMain.cpp" />
     <ClCompile Include="..\src\UnicodeTest.cpp" />
-    <ClCompile Include="..\src\Utilities.cpp" />
+    <ClCompile Include="..\src\PolyLineTest.cpp" />

```

Comment:
Thanks - this should be addressed in recent commits, as well as missing units on Linux cmake project (which I committed blindly; if I missed something there Travis should catch it)